stellarphot/settings/tests/test_models.py

```diff
@@ -213,8 +226,8 @@ def test_name_unicode_is_ok(self, model, settings):
 @pytest.mark.parametrize(
     "model,settings",
     [
-        [Camera, TEST_CAMERA_VALUES.copy()],
-        [Observatory, TEST_OBSERVATORY_SETTINGS.copy()],
+        [Camera, TEST_CAMERA_VALUES],
+        [Observatory, TEST_OBSERVATORY_SETTINGS],
```

Comment:
I'm assuming you are using the deep copies you created in lines 39-48 here so you are not copying them.  Maybe?

Reply:
yep